addons/point_of_sale/static/src/app/components/order_widget/order_widget.xml

```diff
@@ -1,75 +0,0 @@
-                    <div t-if="props.generalCustomerNote"
-                        t-attf-class="{{ props.screenName == 'ReceiptScreen' ? 'mt-1 bg-opacity-75 p-0' : 'customer-note w-100 p-2 mt-2 rounded text-break text-bg-warning bg-opacity-25 mt-0'}}">
-                        <div class="row flex-wrap w-100 m-0"> <!-- Add w-100 and remove margins -->
-                            <div class="col-auto px-1">
-                                <i class="fa fa-sticky-note me-1 fa-lg" role="img"
-                                    aria-label="Customer Note" title="Customer Note"/>
-                            </div>
-                            <div class="col ps-0">
-                                <t t-foreach="props.generalCustomerNote.split('\n')" t-as="subNote"
-                                    t-key="subNote_index">
-                                    <div class="d-inline text-break">
-                                        <t t-esc="subNote"/><br/>
-                                    </div>
-                                </t>
-                            </div>
-                        </div>
-                    </div>
-                    <div t-if="props.internalNote"
-                        class="internal-note-container d-flex gap-2">
-                        <t t-foreach="props.internalNote?.split?.('\n') or []" t-as="note"
-                            t-key="note_index">
-                            <li t-if="note.trim() !== ''" t-esc="note"
-                                class="internal-note badge mt-1 p-2 rounded-pill bg-info text-info bg-opacity-25"
-                                style="font-size: 0.85rem;"/>
-                        </t>
-                    </div>
-                </div>
-                <t t-set="taxTotals" t-value="props.taxTotals"/>
-                <div t-if="taxTotals" class="order-summary d-flex flex-column gap-1 px-3 py-2 border-top fw-bolder lh-sm">
-                    <div t-if="taxTotals.has_tax_groups"
-                         class="tax-info subentry d-flex justify-content-between w-100 fs-6 text-muted"
-                    >
-                        Taxes
-                        <div id="order-widget-taxes">
-                            <span t-esc="formatMonetary(taxTotals.tax_amount_currency, {currencyId: taxTotals.currency_id})"
-                                  class="tax"/>
-                        </div>
-                    </div>
-                    <div t-if="'cash_rounding_base_amount_currency' in taxTotals"
-                         class="subentry d-flex justify-content-between w-100 fs-6 text-muted"
-                    >
-                        Rounding
-                        <div id="order-widget-rounding">
-                            <span t-esc="formatMonetary(taxTotals.cash_rounding_base_amount_currency, {currencyId: taxTotals.currency_id})"
-                                  class="rounding"/>
-                        </div>
-                    </div>
```

Comment:
Looks like the new `addons/point_of_sale/static/src/app/components/order_display/order_display.xml` file doesn't take into account these lines.

Reply:
In order_receipt